docs/en/observability/synthetics-troubleshooting.asciidoc

```diff
@@ -0,0 +1,45 @@
+[[synthetics-troubleshooting]]
+= Troubleshooting Synthetics
+
+++++
+<titleabbrev>Troubleshooting</titleabbrev>
+++++
+
+[discrete]
+[[synthetics-troubleshooting-local-debugging]]
+= Local debugging
+
+Use the `DEBUG=synthetics` environment variable to capture Synthetics agent logs when
+running Synthetic tests locally with the <<synthetics-command-reference,Synthetics CLI>>.
+
+[discrete]
+[[synthetics-troubleshooting-common-issues]]
+= Common issues
+
+[discrete]
+[[synthetics-troubleshooting-missing-api-key]]
+== Missing API key
+
+Missing / incorrect API key (should get notification, but how to regenerate if not getting results)
+
+[discrete]
+[[synthetics-troubleshooting-no-agent-running]]
+== No Agent running
+
+No Agent running with the Private Location Agent Policy (so no results)
```

Comment:
As you have written, if there are no Agents configured to run against the Agent Policy, those monitors configured to run on that Private Location will not be running and you will get no results.

If you have created an Agent Policy already and attempt to assign it to a Private Location (under Synthetics > Settings > Private Locations), you will see a notification that there are no Agents attached:

![image](https://github.com/elastic/observability-docs/assets/70908889/abe40ddf-c0ed-4a91-9392-295f6cac6c09)

If a Private Location is created using such a policy, this will also be shown under Synthetics > Settings > Private Locations:

![image](https://github.com/elastic/observability-docs/assets/70908889/68985656-c548-4f99-bbe0-c4f1fa566bba)

Reply:
Ok, I attempted a first draft of this one, too. I'm not sure if the framing is 100% correct so I'm very open to suggestions! 

Here's the preview link: [No results from a monitor configured to run on a Private Location](https://observability-docs_3081.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-troubleshooting.html#synthetics-troubleshooting-no-agent-running)

Reply:
Thanks Colleen, LGTM